reproducer.py

```
@@ -0,0 +1,60 @@
+import warp as wp
+
+# CPU-mode reproducer adapted from your CUDA snippet
+
+def main():
+    device = wp.get_device("cpu")
+    try:
+        wp.load_module(device=device)
+    except Exception:
+        # load_module may be unnecessary on CPU; ignore failures
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)